apps/dotcom/client/src/tla/components/TlaFileMenu/TlaFileMenu.tsx

```diff
@@ -91,27 +90,29 @@ export function TlaFileMenu({
 	}, [fileId, addDialog])
 
 	const isOwner = useIsFileOwner(fileId)
+
+	const copyLinkMsg = useMsg(messages.copyLink)
+	const renameMsg = useMsg(messages.rename)
+	const duplicateMsg = useMsg(messages.duplicate)
+	const deleteOrForgetMsg = useMsg(isOwner ? messages.delete : messages.forget)
+	const fileSubmenuMsg = useMsg(messages.file)
```

Comment:
btw, we can alternatively allow `label` to accept ReactNode's and then you can use `<F>` instead of `useMsg` which is a little bit more annoying

Reply:
This is a bit of a mess, as the translation keys are designed to take either strings or objects with different strings for different contexts (ie on mobile, or in the context menu, use this alternative label for an action item). We might not need to connect these two systems, but could do